### ISSUE TITLE
Improve banner design and usefulness

### ISSUE THREAD

**Issue by machawk1** [oduwsdl/ipwb#449]
<img width="603" alt="screen shot 2018-07-20 at 9 55 17 pm" src="https://user-images.githubusercontent.com/2514780/43031050-f02863aa-8c67-11e8-8c07-62944961450d.png">

The current replay banner (pictured) has a few issue that should be improved:

- [x] Datetime is an awkward, long, numerical string
- [x] URI-R and datetime are text _editable_ text fields
- [x] Alignment of "Archived at:" and datetime has a lot of awkwardly unused space

I know https://github.com/oduwsdl/reconstructive has much more capability than this. Can we look into taking advantage of this capability, @ibnesayeed?

**Comment by ibnesayeed** [comment]
The improved banner has not been pushed into Reconstructive repo yet. It needs more work to be functional.

**Comment by machawk1** [comment]
Noted @ibnesayeed. Per #144, we are planning on providing a means of navigating between mementos. Perhaps this will be an opportunity to improve the aesthetics and interaction of ipwb's usage of reconstructive.

**Comment by ibnesayeed** [comment]
PR #462 has more than what was enlisted in the OP.

**Comment by ibnesayeed** [comment]
Closing, as this is done!
